File: Npc/11002057.cs
using Maple2.Model.Enum;
using Maple2.Script.Npc;

/// <summary>
/// 11002057: Juniper
/// </summary>
public class _11002057 : NpcScript {
    internal _11002057(INpcScriptContext context) : base(context) {
        Id = 60;
    }

    public override bool Next(int selection = 0) {
        switch (Id) {
            case 0:
                // $script:1219175006000651$ 
                // - Can I help you?
                return true;
            case 60:
                // $script:1219175006000654$ 
                // - You don't live here, do you? This shop is locals only. Please shop elsewhere.
                return true;
            default:
                return true;
        }
    }
}

File: Npc/11002057.py
""" 11002057: Juniper """
from npc_api import Option, Script
import random


class Main(Script):
    def first(self) -> int:
        return 60

    def select(self) -> int:
        return 0

    def __0(self, pick: int) -> int:
        # $script:1219175006000651$
        # - Can I help you?
        return None # TODO

    def __60(self, pick: int) -> int:
        # $script:1219175006000654$
        # - You don't live here, do you? This shop is locals only. Please shop elsewhere.
        return -1

    def button(self) -> Option:
        if (self.state, self.index) == (60, 0):
            return Option.CLOSE
        return Option.NONE
